.github/CODEOWNERS

```diff
@@ -1,4 +1,5 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-*                                    @MetaMask/devs
+*  @MetaMask/wallet-api-platform-engineers
+*  @MetaMask/tech-writers
```

Comment:
Nit: I would assume that GitHub would combine the two together here but you should be able to add both on one line: 

``` suggestion
*  @MetaMask/wallet-api-platform-engineers @MetaMask/tech-writers
```